### PR TITLE
Product review

### PR DESCRIPTION
**Pull request by ShBr2k**
Upd API tests.

### CODE REVIEW

**Review by SET-SKYNET — APPROVED** [review]

File: src/main/java/test/roman/TestAPIPS.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                .body("prestashop.customer.lastname", CoreMatchers.is(strValuesCust[1]))
+                .body("prestashop.customer.email", CoreMatchers.is(strValuesCust[2]))
+                .body("prestashop.customer.birthday", CoreMatchers.is(strValuesCust[3]));
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check API's status code after adding customer with existing value of login (mail)")
+    @Test(priority = 4)
+    public void checkAPIStatusCodeAfterAddWrongLoginCust() {
```
Comment (ostret) [inline src/main/java/test/roman/TestAPIPS.java:78]: Is this expected behaviour or a defect?

File: src/main/java/test/roman/TestAPIPS.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        Assert.assertFalse(getStatusCode == 201);
+        System.out.println("API return status code: " + getStatusCode + ". New customer with wrong password not added.\n");
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check count of customers after adding customer with wrong value of password (<5 chars)")
+    @Test(priority = 7)
+    public void checkCountCustomersAfrerAddWrongPassCust() {
+        Assert.assertTrue(CountCustBeforeAdding == CountCustAfterAdding);
```
Comment (ostret) [inline src/main/java/test/roman/TestAPIPS.java:94]: there is no direct action in this test
if test before it (that has action) fails - then this one makes no sense
move this validation step to the test above

File: src/main/java/test/roman/TestAPIPS.java (hunk 1 above)
Comment (ShBr2k) [reply src/main/java/test/roman/TestAPIPS.java:78]: This is a defect of backend.

File: src/main/java/test/roman/TestAPIPS.java (hunk 2 above)
Comment (ShBr2k) [reply src/main/java/test/roman/TestAPIPS.java:94]: Done, its fixed. A new request created.

**Comment by ShBr2k** [comment]
A new request created.

**Pull request closed without merging**